.github/workflows/haskell.yml

```diff
@@ -47,7 +47,6 @@ jobs:
       matrix:
         cabal: ["3.10"]
         ghc:
-          - "8.4.4"
```

Comment:
Can we restore this? I think the shakespeare issue should be fixed by upstream hackage revision

Reply:
Restored.

Reply:
And all tests pass  :tada: